core/access_api.php

```diff
@@ -509,41 +509,40 @@ function access_has_bug_level( $p_access_level, $p_bug_id, $p_user_id = null ) {
 		return false;
 	}
 
+	# Check the requested access level, shotcut to fail if not satisfied
 	$t_project_id = bug_get_field( $p_bug_id, 'project_id' );
-	$t_bug_is_user_reporter = bug_is_user_reporter( $p_bug_id, $p_user_id );
 	$t_access_level = access_get_project_level( $t_project_id, $p_user_id );
+	if( !access_compare_level( $t_access_level, $p_access_level ) ){
+		return false;
+	}
 
-	# check limit_Reporter (Issue #4769)
-	# reporters can view just issues they reported
-	$t_limit_reporters = config_get( 'limit_reporters', null, $p_user_id, $t_project_id );
-	if( $t_limit_reporters && !$t_bug_is_user_reporter ) {
-		# Here we only need to check that the current user has an access level
-		# higher than the lowest needed to report issues (report_bug_threshold).
-		# To improve performance, esp. when processing for several projects, we
-		# build a static array holding that threshold for each project
-		static $s_thresholds = array();
-		if( !isset( $s_thresholds[$t_project_id] ) ) {
-			$t_report_bug_threshold = config_get( 'report_bug_threshold', null, $p_user_id, $t_project_id );
-			if( empty( $t_report_bug_threshold ) ) {
-				$s_thresholds[$t_project_id] = NOBODY;
-			} else {
-				$s_thresholds[$t_project_id] = access_threshold_min_level( $t_report_bug_threshold ) + 1;
-			}
-		}
-		if( !access_compare_level( $t_access_level, $s_thresholds[$t_project_id] ) ) {
+	# If the level is met, we need to still verify access to the issue
```

Comment:
<del>we need to still verify access to the issue</del> 
--> `we still need to verify that user has access to the issue`